pkg/common/event/interface.go

```diff
@@ -32,6 +32,8 @@ type Event interface {
 	IsPaused() bool
 	// GetLen returns the number of rows in the event.
 	Len() int32
+	// GetIsRedo returns true if the event is handled by redo dispatcher
+	GetIsRedo() bool
```

Comment:
It is strange to add `GetIsRedo` to `Event` interface. You can add a new message topic in event collector side which handle messages for redo dispatchers. And at event broker side choose the  message topic according to the dispatcher type.